<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Директория:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QComboBox" name="sortBox"/>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Имя:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="findLine"/>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Сортировать по:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="4">
    <widget class="QComboBox" name="browseBox"/>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>&amp;Найти</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="5">
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item row="0" column="0" rowspan="3">
    <widget class="QTreeView" name="treeView"/>
   </item>
   <item row="3" column="5">
    <widget class="QPushButton" name="debugButton">
     <property name="text">
      <string>Debug</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="5">
    <widget class="QLabel" name="bottomLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>findLine</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButton</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>472</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>765</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browseBox</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>pushButton</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>754</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>758</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
